airbyte-webapp/src/components/ui/Link/Link.tsx

```diff
@@ -0,0 +1,36 @@
+import { PropsWithChildren } from "react";
+// eslint-disable-next-line no-restricted-imports
+import { Link as ReactRouterLink, To } from "react-router-dom";
+
+import { getLinkClassNames } from "./getLinkClassNames";
+
+export interface LinkProps {
+  className?: string;
+  opensInNewTab?: boolean;
+  variant?: "default" | "primary";
+}
+
+interface InternalLinkProps extends LinkProps {
+  to: To;
+}
+
+export const Link: React.FC<PropsWithChildren<InternalLinkProps>> = ({
+  children,
+  className,
+  to,
+  opensInNewTab = false,
+  variant = "default",
+  ...props
+}) => {
+  return (
+    <ReactRouterLink
+      {...props}
+      className={getLinkClassNames({ className, variant })}
+      rel={opensInNewTab ? "noopener noreferrer" : undefined}
```

Comment:
I don't think we need this here, since we'd always have internal links only in this component. (If we decide to keep, I think the same as my comment above applies).

Reply:
Agreed, addressed in https://github.com/airbytehq/airbyte/pull/22881/commits/73704a4b477e376e51790b28162da90b05e0deeb